### DISCUSSION TITLE
Missing Reference

### DISCUSSION THREAD

**Discussion by SamuelHadsall** [vanjarosoftware/Vanjaro.Platform#2048] [Q&A]
I'm working on creating my local dev environment using the Vanjaro Platform and then merging in the src solution. When I open the solution in Visual Studio it's erroring on system.web.http because the reference is broken in all of the projects. 

Is there a way to fix this reference for all projects?

Am I missing a step before I can build the project?

@meetmandeep @narensingh008 @Gaganbird

**Comment by meetmandeep** [comment]
Have you seen the Setup Webinar? It has step by step instructions: https://www.youtube.com/watch?v=xuxY93seOFc

**Reply by SamuelHadsall** [reply]
Yes, the only difference is I don't use nvquicksite. Is that a requirement? 

My steps are:
Create IIS site with app ppol permissions
Create db, with user as db owner
Install Vanjaro
Pull Develop from Vanjor repo
Copy src files to Website folder
Ran Build in debug

@meetmandeep

**Reply by meetmandeep** [reply]
Did you follow the video? It does not use nvquicksite. We use Vanjaro Setup Installer to setup a dev site.
